src/main/scala/beam/agentsim/agents/household/HouseholdActor.scala

```diff
@@ -459,7 +462,7 @@ object HouseholdActor {
       // Pipe my cars through the parking manager
       // and complete initialization only when I got them all.
       Future
-        .sequence(vehicles.values.map { veh =>
+        .sequence(vehicles.filter(_._2.beamVehicleType.automationLevel <= 3).values.map { veh =>
```

Comment:
only non fully automated cars go through this? what happens with higher automation level?

Reply:
after reversing the filter, it's there to only complete intialization for CAVs because ATM that is all the householdActor managers. the non-CAV vehicles are now managed by HouseholdFleetManager